library/gen_x12_837.inc.php

```diff
@@ -1076,15 +1078,16 @@ function gen_x12_837($pid, $encounter, &$log, $encounter_claim = false)
             "*" . $claim->cptNDCUOM($prockey) .
             // Note: 5010 documents "ME" (Milligrams) as an additional unit of measure.
             "~\n";
+
+            // Segment REF (Prescription or Compound Drug Association Number) omitted.
         }
 
-    // Segment REF (Prescription or Compound Drug Association Number) omitted.
 
     // Loop 2420A, Rendering Provider (service-specific).
     // Used if the rendering provider for this service line is different
     // from that in loop 2310B.
-    //
-        if ($claim->providerNPI() != $claim->providerNPI($prockey)) {
+    // commented block below temporarily since this is removed in 5010
```

Comment:
If this is truly not needed for 5010 and no longer used, then prob best to completely remove it.

Reply:
ok, will do, thanks

Reply:
asked duc tran for current 5010 specs since he has a relationship or possibly even works for the X12 gang